README.md

```diff
@@ -22,6 +22,20 @@ The above steps install PyTorch for CPU only. In order to install a cuda-enabled
 python -m pip install torch==2.0.1 --index-url https://download.pytorch.org/whl/cu118
 ```
 
+Use the following commands to install FAENet:
+
+```bash
+pip install git+https://github.com/RolnickLab/ocp.git@uncertainty-depfaenet
```

Comment:
I set up things to install repos also through the pyproject.toml. I can change this.

Reply:
Update: I will not do this yet because I am not sure I will be able to properly test whether it'll be correct. Not a big deal anyway.